test/sequential/sequencerStaking.rewards.test.ts

```diff
@@ -90,7 +95,7 @@ it("Sequencer budget is set when initializing issuance config", async () => {
 it("Sequencers get paid on every session BUT only when they submit valid updates ( Succeeded extrinsics )", async () => {
   await Sudo.batchAsSudoFinalized(Assets.FinalizeTge(), Assets.initIssuance());
   const disputePeriodLength = (
-    await Rolldown.disputePeriodLength(chainArb)
+    await Rolldown.disputePeriodLength(chainEth)
```

Comment:
why this?

Reply:
I noticed that this test works with the Ethereum chain.